scripts/junit_report.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-console */
+
+const semver = require('semver')
+const { execSync } = require('child_process')
+
+function uploadJUnitXMLReport () {
+  if (semver.lt(process.version, '10.24.1')) {
+    console.log('Node version incompatible with @datadog/datadog-ci. Skipping step.')
+    return
+  }
+  if (process.env.CIRCLE_PR_NUMBER) {
+    console.log('Running in a fork. Skipping step.')
```

Comment:
inspired by https://circleci.com/blog/managing-secrets-when-you-have-pull-requests-from-outside-contributors/#:~:text=To%20enable%20CircleCI%20for%20forked,builds%20from%20forked%20pull%20requests